pubspec.yaml

```diff
@@ -29,7 +29,7 @@ dependency_overrides:
  over_react:
    git:
      url: https://github.com/Workiva/over_react
-     ref: 3.1.0-wip
+     ref: CPLAT-6157-pointer-event-simulator-alias
```

Comment:
You can go back to 3.1.0-wip now that https://github.com/Workiva/over_react/pull/307 has merged.